src/main/java/cms/gov/madie/measure/resources/BundleController.java

```diff
@@ -30,18 +31,29 @@ public class BundleController {
   public ResponseEntity<String> getMeasureBundle(
       @PathVariable String measureId,
       Principal principal,
-      @RequestHeader("Authorization") String accessToken) {
+      @RequestHeader("Authorization") String accessToken,
+      @RequestParam(required = false, defaultValue = "calculation", name = "bundleType")
+          String bundleType) {
     Optional<Measure> measureOptional = measureRepository.findById(measureId);
-    log.info(
-        "User [{}] is attempting to create a new measure bundle for [{}]",
-        principal.getName(),
-        measureId);
+
+    if ("calculation".equalsIgnoreCase(bundleType)) {
+      log.info(
+          "User [{}] is attempting to create a new measure bundle for [{}]",
+          principal.getName(),
+          measureId);
+    } else {
+      log.info("User [{}] is attempting to export measure [{}]", principal.getName(), measureId);
+    }
     if (measureOptional.isEmpty()) {
       throw new ResourceNotFoundException("Measure", measureId);
     }
     Measure measure = measureOptional.get();
     ControllerUtil.verifyAuthorization(principal.getName(), measure);
 
-    return ResponseEntity.ok(bundleService.bundleMeasure(measure, accessToken));
+    if ("calculation".equalsIgnoreCase(bundleType)) {
+      return ResponseEntity.ok(bundleService.getMeasureBundleForCalculation(measure, accessToken));
```

Comment:
method name should be just `getMeasureBundle` with `bundleType` parameter

Reply:
@adongare So I should change getMeasureBundleForCalculation to getMeasureBundle right?